changelog/cache-generated-test-config.dd

```diff
@@ -0,0 +1,3 @@
+cache the generated test main file: dub_test_root.d
```

Comment:
As this is part of the official changelog, It should be a bit more formal and abstract.
Have a look at the currently generated preview changelog:

http://dtest.dlang.io/artifact/website-f7458db425b639e3c2de12c651581f2157d84535-a2c0961de7a30ba66a2edcbbf5b69d6a/web/changelog/pending.html#cache-generated-test-config

Reply:
Thank you for the review.

I try a different phrasing, unfortunate I am not a native english speaker, maybe you have a better suggestion?